app/konnect/network-resiliency.md

```diff
@@ -71,6 +71,10 @@ configuration.
 Whenever a connection is re-established with the control plane, it pushes the latest 
 configuration to the data plane node. It doesn't queue up or try to apply older changes.
 
+### What happens if there are connectivity issues with the Mesh global or zone control planes?
+
+If the global {{site.mesh_product_name}} control plane and zone control plane become disconnected, configuration can't travel between them. You can see [Failure modes](/mesh/latest/production/deployment/multi-zone/#failure-modes) in the {{site.mesh_product_name}} documentation for more details on what happens when there are {{site.mesh_product_name}} connectivity issues.
```

Comment:
```suggestion
If the global {{site.mesh_product_name}} control plane and zone control plane become disconnected, and configuration can't sync between them. You can see [Failure modes](/mesh/latest/production/deployment/multi-zone/#failure-modes) in the {{site.mesh_product_name}} documentation for more details on what happens when there are {{site.mesh_product_name}} connectivity issues.
```

Reply:
I think? This is what you meant. 

Is the purpose to redirect them to the failure modes doc? Then I would do it sooner. 

###  What happens if there are connectivity issues with the Mesh global or zone control planes?

*Insert thing that is explicitly important to a Konnect user* For more information you can find a list of failure modes in the Mesh [failure modes](https://deploy-preview-6098--kongdocs.netlify.app/mesh/latest/production/deployment/multi-zone/#failure-modes) documentation.